testsuite/layers/pom.xml

```diff
@@ -1004,6 +1004,42 @@
                         </configuration>
                     </execution>
                     <!-- wildfly layers -->
+                    <execution>
+                        <id>batch-jberet-provisioning</id>
+                        <goals>
+                            <goal>provision</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <install-dir>${layers.install.dir}/bean-validation</install-dir>
```

Comment:
Your install-dir is not correct bean-validation ==> batch-jberet.

Reply:
Fixed